MVP/README.md

```diff
@@ -0,0 +1,76 @@
+# Platform-CI MVP
+
+## What's MVP?
+
+Minimal Viable Product (MVP) is a minimized set of Jenkins job definition (and supporting files) on Platform CI:
+
+  - for End-to-End workflow changes (batch update) via **Template / Default YAML**
+  - for create/update/scale tests (by individuals) via **Test / Job YAML**
+
+## How to create a CI job with MVP?
+
+1. ####Get MVP files
+
+  >$ git clone https://github.com/RHQE/platform-ci.git
+
+  >$ cd platform-ci/MVP
+
+1. ####Tweak [sample_job.yaml](/MVP/sample_job.yaml/) with your own test parameters
+
+    *Job YAMLs store parameters which are specific for your testing jobs. It's recommended to maintain them in your team git repo.*
+
```

Comment:
So what is the expectation here ?

Teams maintain their yaml in their git repo and platform-ci git repo maintains the template/defaults and every time the component owner needs to rebuild jobs then they copy component yaml from their repo to platform-ci locally and run jjb to have their jobs reconfigured ?